nix/pkgs/default.nix

```diff
@@ -155,7 +154,7 @@ let
   plutus-haddock-combined =
     let
       haddock-combine = pkgs.callPackage ../lib/haddock-combine.nix {
-        ghc = haskell.project.pkg-set.config.ghc.package;
+        ghc = haskell.projectAllHaddock.pkg-set.config.ghc.package;
```

Comment:
Uh, don't we need to pass this on line 162, mostly?

Reply:
Oh no, ignore me, it's inside `haskell`!

Reply:
@michaelpj I'm not totally sure what you mean; do you mind elaborating a bit? - I just changed this one here to refer to the ghc from the haddock-configured packages; I don't believe it matters, but it seemed nice to be consistent? I'm probably misunderstanding :)

Reply:
No, I was just confused!